DocumentsFromSnapshotMigration/build.gradle

```diff
@@ -170,6 +170,8 @@ task slowTest(type: Test) {
 jacocoTestReport {
     dependsOn slowTest
     reports {
+        xml.required = true
+        xml.destination file("${buildDir}/reports/jacoco/test/jacocoTestReport.xml")
         html.required = true
         html.destination file("${buildDir}/reports/jacoco/test/html")
     }
```

Comment:
Can we update this in the all projects configuration?

Reply:
The rest of them should be in sync w/ this standard already; this was the straggler.

Reply:
This looks like something we can wrap in subProjects{} within the top level build.gradle and remove from all other projects in a future PR